src/main.rs

```diff
@@ -14,46 +14,6 @@ use cli::{ResizeStrategy, Swww};
 
 fn main() -> Result<(), String> {
     let swww = Swww::parse();
-    if let Swww::Init {
-        no_daemon, format, ..
-    } = &swww
-    {
-        eprintln!(
-            "DEPRECATION WARNING: `swww init` IS DEPRECATED. Call `swww-daemon` directly instead"
-        );
-        match is_daemon_running() {
-            Ok(false) => {
-                let socket_path = get_socket_path();
-                if socket_path.exists() {
-                    eprintln!(
-                        "WARNING: socket file {} was not deleted when the previous daemon exited",
-                        socket_path.to_string_lossy()
-                    );
-                    if let Err(e) = std::fs::remove_file(socket_path) {
-                        return Err(format!("failed to delete previous socket: {e}"));
-                    }
-                }
-            }
-            Ok(true) => {
-                return Err("There seems to already be another instance running...".to_string())
-            }
```

Comment:
I think I actually need to move this logic into the daemon, right?

Reply:
It's already there, actually.

Reply:
It's already there, actually :).